tokio-timer/src/atomic.rs

```diff
@@ -0,0 +1,56 @@
+//! Implementation of an atomic u64 cell. On 64 bit platforms, this is a
+//! re-export of `AtomicU64`. On 32 bit platforms, this is implemented using a
+//! `Mutex`.
+
+pub(crate) use self::imp::AtomicU64;
+
+#[cfg(target_pointer_width = "64")]
```

Comment:
This needs to be `#[cfg(target_has_atomic = "64")]`, not `#[cfg(target_pointer_width = "64")]`, but since `feature(cfg_target_has_atomic)` is not stable, will use an alternative.

Reply:
Fixed in 3e86b76